c2rust/Cargo.toml

```diff
@@ -1,34 +1,34 @@
 [package]
-name = "c2rust"
-description = "C to Rust translation, refactoring, and cross-checking"
-version = "0.16.0"
 authors = ["The C2Rust Project Developers <c2rust@immunant.com>"]
-license = "BSD-3-Clause"
-homepage = "https://c2rust.com/"
-repository = "https://github.com/immunant/c2rust"
-edition = "2021"
 categories = ["development-tools", "development-tools::ffi", "command-line-utilities"]
+default-run = "c2rust"
+description = "C to Rust translation, refactoring, and cross-checking"
+edition = "2021"
+homepage = "https://c2rust.com/"
 keywords = ["transpiler", "migration", "translation", "c"]
+license = "BSD-3-Clause"
+name = "c2rust"
 readme = "README.md"
-default-run = "c2rust"
+repository = "https://github.com/immunant/c2rust"
+version = "0.16.0"
```

Comment:
Please don't re-order these.  While a standard order would be nice, alphabetical is not the right one, as `name = "c2rust"` makes sense to be first.  And if we did, it should be in a separate PR.